EmailMessageTrigger.trigger

```diff
@@ -0,0 +1,25 @@
+trigger EmailMessageTrigger on EmailMessage (before insert) {
+	//get the list of incoming messages
+    List<EmailMessage> incomingMessages = Trigger.new;
+    System.debug('Size of incoming messages ' + incomingMessages.size());
+    System.debug('Messaging ' + incomingMessages);
```

Comment:
@jivan0608 
If you want to debug an object or collection of objects it's better to use
JSON.serialize(your collection)